common/src/main/java/ch/admin/bag/covidcertificate/common/qr/QrScanFragment.kt

```diff
@@ -78,6 +78,12 @@ abstract class QrScanFragment : Fragment() {
 	private var cameraPermissionState = CameraPermissionState.REQUESTING
 	private var cameraPermissionExplanationDialog: CameraPermissionExplanationDialog? = null
 	private var isTorchOn: Boolean = false
+	private val autoFocusClockLiveData = liveData<Unit> {
+		while (true) {
```

Comment:
Instead of `while (true)` it's better to check for `currentCoroutineContext().isActive` to see if the coroutine that subscribed to this livedata is still active